icons.yml

```diff
@@ -7,390 +7,327 @@ mappings:
   - subclass: nightclub
     iconName: music-11
-    color: '#8A9CF6'
+    color: '#E93865'
   - subclass: prison
     iconName: prison-11
-    color: '#8A9CF6'
+    color: '#495063'
   - subclass: sports_centre
     iconName: basketball-11
-    color: '#8A9CF6'
+    color: '#5C6F84'
   - subclass: fuel
     iconName: fuel-11
-    color: '#8A9CF6'
-  - subclass: marina
-    iconName: harbor-11
-    color: '#8A9CF6'
-  - subclass: college
-    iconName: college-11
-    color: '#A3A3A3'
-  - subclass: books
-    iconName: library-11
-    color: '#FFC324'
-  - subclass: library
-    iconName: library-11
-    color: '#FFC324'
-  - subclass: university
-    iconName: college-11
-    color: '#A3A3A3'
-  - subclass: beauty
-    iconName: shop-11
-    color: '#FFC324'
+    color: '#5C6F84'
   - subclass: gift
     iconName: gift-11
-    color: '#000'
+    color: '#F3C600'
   - subclass: recycling
     iconName: recycling-11
-    color: '#000'
+    color: '#5C6F84'
   - subclass: mall
     iconName: commercial-11
-    color: '#000'
+    color: '#F3C600'
   - subclass: marketplace
     iconName: commercial-11
-    color: '#000'
-  - subclass: townhall
-    iconName: town-hall-11
-    color: '#8A9CF6'
-  - subclass: courthouse
-    iconName: town-hall-11
-    color: '#8A9CF6'
-  - subclass: accessories
-    iconName: shop-11
-    color: '#FFC324'
-  - subclass: charity
-    iconName: shop-11
-    color: '#FFC324'
-  - subclass: general
-    iconName: shop-11
-    color: '#FFC324'
-  - subclass: second_hand
-    iconName: shop-11
-    color: '#FFC324'
-  - subclass: doityourself
-    iconName: shop-11
-    color: '#FFC324'
-  - subclass: wine
-    iconName: alcohol-shop-11
-    color: '#FFC324'
+    color: '#F3C600'
   - subclass: video_games
     iconName: gaming-11
-    color: '#FFC324'
+    color: '#F3C600'
   - subclass: toys
     iconName: gaming-11
-    color: '#FFC324'
+    color: '#F3C600'
   - subclass: gate
     iconName: barrier-11
-    color: '#000'
+    color: '#5C6F84'
   - subclass: cycle_barrier
     iconName: barrier-11
-    color: '#000'
+    color: '#5C6F84'
   - subclass: stile
     iconName: barrier-11
-    color: '#000'
+    color: '#5C6F84'
   - subclass: shelter
     iconName: shelter-11
-    color: '#000'
+    color: '#5C6F84'
   - subclass: sally_port
     iconName: ranger-station-11
-    color: '#000'
+    color: '#5C6F84'
   - subclass: basin
     iconName: dam-11
-    color: '#000'
+    color: '#76BC54'
   - subclass: reservoir
     iconName: dam-11
-    color: '#000'
+    color: '#76BC54'
   - subclass: monument
     iconName: monument-11
-    color: '#8A9CF6'
-  - subclass: castle
-    iconName: castle-11
-    color: '#8A9CF6'
-  - subclass: ruins
-    iconName: castle-11
-    color: '#8A9CF6'
+    color: '#5C6F84'
   - subclass: airport
     iconName: airport-11
-    color: '#5CA6FB'
+    color: '#8665E7'
+  - subclass: athletics
+    iconName: pitch-11
+    color: '#5C6F84'
+  - subclass: pitch
+    iconName: pitch-11
```

Comment:
the `pitch` picto looks like an emergency exit one, can we switch to the `soccer` one (while we enhance the tile schema to better handle sports item) ?